### PR TITLE
Updated CDK version for the new BP

### PR DESCRIPTION
**Pull request by hmuthusamy**
*Issue #, if available:*
updated CDK version for new blueprints for CDK accelerator
*Description of changes:*

updated CDK version for new blueprints for CDK accelerator
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by elamaran11 — APPROVED** [review]
LGTN

File: aws-cdk-eks-app-alarms-to-slack/package.json [hunk 1]
```diff
@@ -13,15 +13,15 @@
...
     "@types/node": "18.11.18",
-    "aws-cdk": "2.114.1",
+    "aws-cdk": "2.132.0",
     "jest": "^29.3.1",
     "ts-jest": "^29.0.3",
     "ts-node": "^10.9.1",
     "typescript": "~4.9.4"
   },
   "dependencies": {
     "@aws-quickstart/eks-blueprints": "^1.5.4",
```
Comment (elamaran11) [inline aws-cdk-eks-app-alarms-to-slack/package.json:23]: Lets also update the blueprints version.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
Minor comment.

File: aws-cdk-eks-app-alarms-to-slack/package.json (hunk 1 above)
Comment (hmuthusamy) [reply aws-cdk-eks-app-alarms-to-slack/package.json]: updated to new BP version which was validated

**Pull request merged**